index.html

```diff
@@ -1,41 +1,156 @@
+        <div class="tonic"><img alt="#" src ="./Work/Tonic2.png"> </div>
+        <div class ="header" id ="title"><h1>Tonic</h1></div>
+        <ul class="details">
+            <li><div class ="canopy"><p>CANOPY</p></div></li>
+            <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+            <li><div class ="spec"><p>Back End Dev</p></div></li>
+            <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+            <li><div class ="spec"><p>2015</p></div></li>
+        </ul>
+        <div class="info"> <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+        <ul class="lang">
+            <li><div class="hcj"><p>html</p></div></li>
+            <li><div class="hcj"><p>css</p></div></li>
+            <li><div class="hcj"><p>javascript</p></div></li>
+        </ul>
+        <div class ="see"><button class ="button"type ="button"><p>See Project</p></button></div>
     </section>
+
+<section class="middle-section">
+    <div class="tonic"><img alt="#" src ="./Work/Multi-post stories.png"> </div>
+    <div class ="header" id ="title"><h1>Multi-Post Stories</h1></div>
+    <ul class="details">
+        <li><div class ="canopy"><p>CANOPY</p></div></li>
+        <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+        <li><div class ="spec"><p>Back End Dev</p></div></li>
+        <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+        <li><div class ="spec"><p>2015</p></div></li>
+    </ul>
+    <div class="info"> <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+    <ul class="lang">
+        <li><div class="hcj"><p>html</p></div></li>
+        <li><div class="hcj"><p>css</p></div></li>
+        <li><div class="hcj"><p>javascript</p></div></li>
+    </ul>
+    <div class ="see"><button class ="button"type ="button"><p>See Project</p></button></div>
+</section>
+
+<section class="about-sec">
+    <div class="header"><h3>About Myself</h3></div>
+    <div class="info"><p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    <div class="connect"><p>LET'S CONNECT</p></div>
+    <ul class ="icons">
+        <li class="cat"><a href="#"><img alt="#" src = "./Icons/cat.png"></a></li>
+        <li class="linkdin"><a href="#"><img alt="#" src = "./Icons/Linkedin icon.png"></a></li> 
+        <li class="p"><a href="#"><img alt="#" src = "./Icons/Peace.png"></a></li>
+        <li class="twitter"><a href="#"><img alt="#" src = "./Icons/Twitter.png"></a></li>
+        <li class="m"><a href="#"><img alt="#" src = "./Icons/M.png"></a></li>    
+    </ul>
+    <div class ="resume"><button class ="buttonr"type ="button"><p>Get my resume</p></button></div>
```

Comment:
- The border of this button as per FIGMA design is `border: 1px solid`
- Change the background color of the button as recommended by previous reviewer. 
![image](https://user-images.githubusercontent.com/61361037/219858157-2c8a4e6e-3a29-4c99-a61e-d02a9f1e6e76.png)

Reply:
- [ ] Great job changing the background color but you still haven't changed the border size and color of the button as the previous reviewer suggested. Kindly change that as well. :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/219865384-86fc409f-c433-4ef3-b7bf-105e4a3a3458.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/219865342-914bc7cc-cf85-4c26-88e0-36f77e9f458c.png)
- [ OPTIONAL ] All the other buttons have the same border styles so I will suggest you apply the same border styles to all the other buttons :+1: